hazelcast/src/main/java/com/hazelcast/map/impl/operation/TryPutOperation.java

```diff
@@ -54,6 +45,6 @@ public void onWaitExpire() {
 
     @Override
     public Object getResponse() {
-        return successful;
+        return true;
```

Comment:
it was like that before, `tryPut` method of record-store which I've removed/cleaned was calling `putInternal` and returning true.